### ISSUE TITLE
F5 debug run of ansible playbooks

### ISSUE THREAD

**Issue by pertoft** [ansible/vscode-ansible#485]
### Problem

F5 run configuration is not working with ansible. 

### Solution

Enable a launch.json configuration for ansible like with python.

**Comment by charltonstanley** [comment]
Yes please!! It's pretty painful trying to develop complex playbooks & roles without this.

**Comment by Tualua** [comment]
Any progress/workaround on this?

**Comment by MaykGyver** [comment]
A fully fledged debugger with breakpoints and stuff for ansible would be nice. Because I keep hitting F5 I helped myself with the [F5 Anything extension](https://marketplace.visualstudio.com/items?itemName=discretegames.f5anything).
